flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/table/ReducingUpsertWriter.java

```diff
@@ -140,7 +141,7 @@ private RowData changeFlag(RowData value) {
         return value;
     }
 
-    private void flush() throws IOException, InterruptedException {
+    private void flushBuffer() throws IOException, InterruptedException {
```

Comment:
nitpick: I wonder if `sinkReducedRecordsInBuffer` is a better name here, since we're not actually flushing the wrapped writer. Just to make it semantically clear that we're still only flushing on checkpoints.

Reply:
Done in [5847885](https://github.com/apache/flink-connector-kafka/pull/17/commits/58478851fd4cf9b33ce352ea2a01d89cf995f170).